Sources/TransferMethod/AddTransferMethodPresenter.swift

```diff
@@ -30,7 +30,7 @@ protocol AddTransferMethodView: class {
     func showBusinessError(_ error: HyperwalletErrorType, _ handler: @escaping () -> Void)
     func showLoading()
     func showProcessing()
-    func showTransferMethodFields(_ fields: [HyperwalletField], _ transferMethodTypeDetail: TransferMethodTypeDetail)
+    func showTransferMethodFields(_ fields: [HyperwalletFieldGroup], _ transferMethodTypeDetail: TransferMethodTypeDetail)
```

Comment:
SwiftLint line length violation - Format it. Also it should be HyperwalletTransferMethodType instead of TransferMethodTypeDetail

Reply:
fixed